recipes/biopet-extractadaptersfastqc/0.1/meta.yaml

```diff
@@ -0,0 +1,37 @@
+# !! THIS FILE WAS AUTOMATICALLY GENERATED BY THE SBT-BIOCONDA PLUGIN !!
+# !!                       DO NOT EDIT MANUALLY                       !!
+
+package:
+  name: biopet-extractadaptersfastqc
+  version: '0.1'
+source:
+  url: https://github.com/biopet/extractadaptersfastqc/releases/download/v0.1/ExtractAdaptersFastqc-assembly-0.1.jar
+  sha256: e060b0da4cfe3b7273242ede54e56f61ef3f733795958022cac92f23b1a956f4
+build:
+  number: 0
+requirements:
+  run:
+  - openjdk >=8,<9
+  - python
+about:
+  home: https://github.com/biopet/extractadaptersfastqc
+  license: MIT
+  summary: ExtractAdaptersFastqc reads which adapter sequences where found from a FastQC raw report.
+  description: |-
+    ExtractAdaptersFastqc reads which adapter sequences where found from a FastQC raw report. These sequences can be used as input for a QC tool such as cutadapt.
+
+    The sequences can be output in plain text format with a newline character as a separator between the sequences. Alternatively the sequences can be output in FASTA format.
+
+    For documentation and manuals visit our [github.io page](https://biopet.github.io/extractadaptersfastqc).
+test:
+  commands:
+  - biopet-extractadaptersfastqc --version
+  - biopet-extractadaptersfastqc --help
+extra:
+  notes: 'biopet-extractadaptersfastqc is a Java program that comes with a custom
+    wrapper shell script. By default ''no default java option'' is set in the wrapper.
+    The command that runs the program is ''biopet-extractadaptersfastqc''. If you
+    want to overwrite it you can specify memory options directly after your binaries.
+    If you have _JAVA_OPTIONS set globally this will take precedence. For example
+    run it with ''biopet-extractadaptersfastqc -Xms512m -Xmx1g''. '
```

Comment:
addding a DOI and/or a bio.tools identifier?

Reply:
This is just a small tool that extracts adapters from a fastqc report. It does not do any analysis. 
It is useful as part of a pipeline that executes fastqc and then runs cutadapt. We used it in the [biopet pipelines](https://github.com/biopet/biopet) in the flexiprep pipeline. But we are now transitioning to [wdl](https://github.com/biowdl/QC) so we are using it in there.

We might publish it as part of publishing the pipeline, but as of now there is no DOI yet.